src/pages/Admin/Admin.tsx

```diff
@@ -410,6 +401,15 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
             )}
           </>
         )}
+        {canBeDeleted(venue) && (
+          <button
+            role="link"
+            className="btn btn-block btn-danger"
+            onClick={() => setShowDeleteModal(true)}
```

Comment:
This will cause re-renders, and so should be extracted into a `useCallback`'d `const` in the main body of the component (before the JSX)

Further Reading:

- https://reactjs.org/docs/hooks-reference.html#usecallback

---

You may also be able to use our custom `useShowHide` hook to solve this in a neater way, by replacing the `useState` above:

- https://github.com/sparkletown/sparkle/blob/staging/src/hooks/useShowHide.ts